packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.ts

```diff
@@ -25,15 +25,20 @@ export class Breadcrumb extends FASTElement {
                 this.slottedBreadcrumbItems.length - 1
             ];
 
-            this.removeLastItemSeparator(lastNode);
+            this.setItemSeparator(lastNode);
             this.setLastItemAriaCurrent(lastNode);
         }
     }
 
-    private removeLastItemSeparator(lastNode: HTMLElement): void {
-        if (lastNode instanceof BreadcrumbItem) {
-            (lastNode as BreadcrumbItem).separator = false;
-        }
+    private setItemSeparator(lastNode: HTMLElement): void {
+        this.slottedBreadcrumbItems.forEach((item: HTMLElement) => {
+            if (item instanceof BreadcrumbItem) {
+                (item as BreadcrumbItem).separator = true;
+            }
+            if (lastNode instanceof BreadcrumbItem) {
```

Comment:
Do you need to do the lastNode check for each iteration of the loop or just once outside the loop?

Reply:
updated